contracts/Aavegotchi/facets/PolygonXGotchichainBridgeFacet.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.1;
+
+import {Aavegotchi} from "../libraries/LibAppStorage.sol";
+import {Modifiers} from "../libraries/LibAppStorage.sol";
+import {LibItems} from "../libraries/LibItems.sol";
+
+contract PolygonXGotchichainBridgeFacet is Modifiers {
+
+    address public layerZeroBridge;
+
+    modifier onlyLayerZeroBridge() {
```

Comment:
@KevinKons I think, We'd better to move this modifier to Modifiers Contract like other modifiers.